docs/eventing/samples/sequence/sequence-reply-to-event-display/README.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Sequence Wired to event-display"
+weight: 20
+type: "docs"
+---
+
+# Using Sequences in series
+
+## Prerequisites
+
+For this example, we'll assume you have set up an `InMemoryChannel`
+as well as Knative Serving (for our functions). The examples use `newbroker`
+namespace, again, if you want to deploy to another Namespace, you will need
+to modify the examples to reflect this.
+If you want to use different type of `Channel`, you will have to modify the
+`Sequence.Spec.ChannelTemplate` to create the appropriate Channel resources.
+
+## Overview
+
+We are going to create the following logical configuration. We create a CronJobSource,
+feeding events to a `Sequence`, then taking the output of that `Sequence` and
+displaying the resulting output.
+
+![Logical Configuration](./sequence-reply-to-event-display.png)
+
+
+## Setup
+
+### Create the Knative Services
+
+Change `newbroker` below to create the steps in the Namespace where you want resources
+created.
+
+```yaml
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: first
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
```

Comment:
we should use the latest serving API ? 

see: https://github.com/knative/docs/pull/1480

Reply:
Ok, I tried switching off the #1480 but then things don't resolve properly. I'll debug more tomorrow, it's starting to get late here.
Side note, I think those need to be v1beta1 instead of v1alpha1?

Reply:
Ok, fixed :)